dist/Data-Dumper/Dumper.xs

```diff
@@ -335,7 +339,7 @@ esc_q_utf8(pTHX_ SV* sv, const char *src, STRLEN slen, I32 do_utf8, I32 useqq)
                 * first byte */
                 increment = (k == 0 && *s != '\0') ? 1 : UTF8SKIP(s);
 
-                r = r + my_sprintf(r, "\\x{%" UVxf "}", k);
+                r = r + my_snprintf(r, "\\x{%" UVxf "}", k);
```

Comment:
I dont think this can be right. my_snprintf() has the signature:

```
len = my_snprintf(ptr, max_len, format, ...);
```

eg there is already an example in the file:

```
len = my_snprintf(tmpbuf, sizeof(tmpbuf), "%" IVdf, SvIV(val));
```